#This message is used to send command to tetra
#Parameter read command

int32 num
---
int32 P0_UART1_BAUDRATE
int32 P1_UART2_BAUDRATE
int32 P2_MODE_SELECT
int32 P3_OUTPORT
int32 P4_LOADCELL_ZERO
int32 P5_LOADCELL_MIN
int32 P6_LOADCELL_GAIN
int32 P7_CONVEYOR_DIR
int32 P8_CONVEYOR_TIMEOUT
int32 P9_FIX_LOADING_START_DELAY
int32 P10_FIX_LOADING_END_DELAY
int32 P11_FIX_UNLOADING_START_DELAY
int32 P12_FIX_UNLOADING_MOTOR_DELAY
int32 P13_FIX_UNLOADING_END_DELAY
int32 P14_LOADING_START_DELAY
int32 P15_LOADING_END_DELAY
int32 P16_UNLOADING_START_DELAY
int32 P17_UNLOADING_END_DELAY
int32 P18_EXPRESS_QUANTITY
int32 P19_TERMINAL_LOADING_BASE
int32 P20_TERMINAL_UNLOADING_BASE

bool command_Result

